cinema/serializers.py

```diff
@@ -68,13 +79,72 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
+        )
+
+
+class TicketSerializer(serializers.ModelSerializer):
+    movie_session = MovieSessionSerializer(many=False)
+
+    def validate(self, attrs):
+        data = super(TicketSerializer, self).validate(attrs)
```

Comment:
in new python version there is no need to explicitly add current class and instance to super(), 'cause it is default now